src/bttrx_wifi.cpp

```diff
@@ -22,6 +22,8 @@ Contact: bt-trx.com, mail@bt-trx.com
 
 #include "bttrx_wifi.h"
 
+extern Preferences preferences;
+
```

Comment:
Same as in bttrx_fsm, please do not use Preferences directly, use the bttrx_control object instead.

Reply:
Also fixed with commit https://github.com/phl0/firmware/commit/888408e3974973b4779f6f34db16dabc96b020a2.